tests/test_interact.py

```diff
@@ -57,6 +57,18 @@ def test_interact_escape(self):
         assert not p.isalive()
         assert p.exitstatus == 0
 
+    def test_interact_escape_None(self):
+        " Return only after Termination when `escape_character=None'. "
+        p = pexpect.spawn('{self.interact_py} --no-escape'.format(self=self),
+                          timeout=5, env=self.env)
+        p.expect('<in >')
+        p.sendcontrol('d')
+        p.expect('<eof>')
+        p.expect_exact('Escaped interact')
```

Comment:
Shouldn't this first check sending the normal escape character, to check that it doesn't escape interact?